rails_event_store_active_record/lib/rails_event_store_active_record/event_repository.rb

```diff
@@ -137,11 +135,7 @@ def detect_pkey_index_violated(e)
 
     def build_event_entity(record)
       return nil unless record
```

Comment:
removing of this check is resulting:

Failures:

  1) RailsEventStoreActiveRecord::EventRepository behaves like event_repository knows last event in stream
     Failure/Error:
       record.event.event_type.constantize.new(
         event_id: record.event.id,
         metadata: serializer.load(record.event.metadata),
         data:     serializer.load(record.event.data)
       )
     
     NoMethodError:
       undefined method `event' for nil:NilClass
     Shared Example Group: :event_repository called from ./spec/event_repository_spec.rb:24

Reply:
Then we need to fix

```
    def last_stream_event(stream_name)
      build_event_entity(
        EventInStream.where(stream: stream_name).order('position DESC, id DESC').first
      )
    end
```

to handle `EventInStream.where(stream: stream_name).order('position DESC, id DESC').first` being `nil` to not call `build_event_entity`

Reply:
this will be just moving condition again UP :) right?

Reply:
done

Reply:
Yep, but it is only needed in one place (I believe) so let's keep it close to that one place only.